driver/driver.go

```diff
@@ -376,13 +402,6 @@ func initConfig(lCfg *config.Driver) (*config.Driver, error) {
 		cfg.URL = DefaultURL
 	}
 
-	if cfg.Project == "" {
```

Comment:
Do you want to remove the rest of the environment handling from here as well?

Reply:
We should not.
Direct user or the driver: CLI, Ferret, depend on this functionality.

The only variable we need to move up to the high level client is "project", because driver is not bound to the project and require project to be specified explicitly.

Reply:
That doesn't make the code clean then as environment handling is partially happening in two different places.